src/openforms/analytics_tools/utils.py

```diff
@@ -0,0 +1,42 @@
+import hashlib
+import json
+import os
+from typing import List, Optional
+
+from django.conf import settings
+
+PATH = os.path.abspath(os.path.dirname(__file__))
+SITE_ID = "SITE_ID"
+DOMAIN_HASH = "DOMAIN_HASH"
+SITE_URL = "SITE_URL"
+
+
+def get_cookies(analytics_tool: str, site_id: Optional[int] = None) -> List[dict]:
+    with open(f"{PATH}/contrib/{analytics_tool}/cookies.json", "r") as f:
+        cookies = json.load(f)
+        if site_id is None:
+            return cookies
+        for cookie in cookies:
+            cookie["name"] = cookie["name"].replace(SITE_ID, str(site_id))
+            cookie["name"] = cookie["name"].replace(
+                DOMAIN_HASH,
+                get_domain_hash(settings.ALLOWED_HOSTS[0], cookie["path"]),
+            )
+        return cookies
```

Comment:
working with a list of `token -> replacements` (or a dict, doesn't really matter) would be a bit more generic and clean, and you can still reuse piwik/piwik pro/matomo through a utility function.